src/makeInfinityAdder.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (number) => {
+    return createFunction(number);
+  };
```

Comment:
I think you have some redundancies with inner functions here. Also pay attention to the redundant empty lines in your code. It doesn't increase readability.